qa/suites/rados/verify/validater/valgrind.yaml

```diff
@@ -8,7 +8,8 @@ overrides:
   ceph:
     conf:
       global:
-        osd heartbeat grace: 80
+        # see https://tracker.ceph.com/issues/65768
+        osd heartbeat grace: 90
```

Comment:
Let's not increase by 10, double it instead to 160.

Reply:
Done.